data/range-input-generator.py

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env python3
```

Comment:
I think it would good to separate the scripts to generate data from where the data is stored. Maybe these scripts could move into a new `scripts` or `hack` director.

Reply:
moved to hack dir